x/incentives/keeper/gauge.go

```diff
@@ -446,13 +446,8 @@ func (k Keeper) GetGaugeByID(ctx sdk.Context, gaugeID uint64) (*types.Gauge, err
 }
 
 // GetGaugeFromIDs returns gauges from gauge ids reference
-func (k Keeper) GetGaugeFromIDs(ctx sdk.Context, refValue []byte) ([]types.Gauge, error) {
+func (k Keeper) GetGaugeFromIDs(ctx sdk.Context, gaugeIDs []uint64) ([]types.Gauge, error) {
 	gauges := []types.Gauge{}
-	gaugeIDs := []uint64{}
-	err := json.Unmarshal(refValue, &gaugeIDs)
-	if err != nil {
-		return gauges, err
-	}
```

Comment:
Unrelated to this PR, but noticed that this method in PR takes in refValue(array of bytes) as parameter. 
I'm guessing this was originally written this way because pagination sends in paginated results as array of bytes, but IMO I think it's weird that a method takes in a parameter not json Unmarshalled, and doing it within the method.

Reply:
If folks think this change should not happen here, or need further discussion upon this change, I could also address this in a seperate PR

Reply:
yep, I think #507 is good to be closed! 
Actually let me make a different PR to migrate the method into grpc_query.go and fix other methods with this as well!